tests/integration/django/dill/leaves/features/steps.py

```diff
@@ -79,9 +79,10 @@ def count_harvesters(step):
 
 @writes_models(Panda)
 def write_pandas(data, field):
-    # It is not necessary to call hashes_data, but it might be present in old
-    # code using the library. Test that it is a no-op in that case.
-    data = hashes_data(data)
+    # It is not necessary to call hashes_data/guess_types, but it might be
+    # present in old code using the library. Test that it is a no-op
+    # in that case.
```

Comment:
I thought about removing it, but decided to keep it in case someone munges up a port.